archivist/assets.py

```diff
@@ -145,9 +148,11 @@ def create(
         # in the method args will overide...
         newprops = _deepmerge({"behaviours": BEHAVIOURS}, props)
         data = self.__query(newprops, attrs)
-        return self.create_from_data(data, confirm=confirm)
+        return self.create_from_data(data, confirm=confirm, exist_ok=exist_ok)
```

Comment:
I don't really like the name of this arg, I feel like it doesn't adequately describe what it does, how about `verify_unique`?

Reply:
Hmm - maybe. I was taking a cue from similar args to os.makedirs - for example...

Reply:
removed...